packages/access-api/package.json

```diff
@@ -28,6 +28,7 @@
     "@web3-storage/access": "workspace:^",
     "@web3-storage/capabilities": "workspace:^",
     "@web3-storage/worker-utils": "0.4.3-dev",
+    "dotenv": "^16.0.3",
```

Comment:
`dotenv` is required by the `build` script, so moving this from devDep to dep fixed gh actions.